src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -32,11 +51,47 @@ public DriveTrain() {
 		leftFollower2.restoreFactoryDefaults();
 	}
 
-	public void drive(double speed, double turn) {
-		robotDrive.arcadeDrive(speed * DriveConstants.SPEED_MULTIPLIER, turn * DriveConstants.TURN_MULTIPLIER);
-	}
 
-	public void park() {
+    public void shiftUp(){shifter.set(Value.kForward);}
 
-	}
+    public void shiftDown(){shifter.set(Value.kReverse);}
+
+    public void compressorStart(){
+        m_compressor.start();
+    }
+
+    public void compressorStop(){
+        m_compressor.stop();
+    }
+
+    public double getYaw(){
+        double x;
```

Comment:
are you sure this is how you get the value from limelimelight, I thought .getDouble() was to choice which pipeline we wanted